docks/build/Dockerfile

```diff
@@ -0,0 +1,38 @@
+## goclang/build
+## a Docker environment to build and test go-clang
+from goclang/base
+maintainer go-clang-devs@github.com/go-clang
+
+env LLVM_VERSION 3.4
+
+## add repositories
+run apt-get install -y software-properties-common && \
```

Comment:
I know that this will look rather ugly but how about doing all the apt-* commands within one RUN?